### PR TITLE
createst: Allow to exclude certain fields

### PR DESCRIPTION
**Pull request by Nancyenos**
Ticket: #4062

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4062

## Ticket
- This is a continuation of PR #997 by  @TharushiJay
-  exclude Non top-level fields fields
- updated readne
- previous pr #2110 
```
createst.py mytest mypcap --exclude-fields dest_port,tcp.ts_max_regions
```
Before:
```
 filter:
    count: 1
    match:
      app_proto: http
      dest_ip: 82.165.177.154
      event_type: flow
      flow.age: 0
      flow.alerted: false
      flow.bytes_toclient: 495
      flow.bytes_toserver: 425
      flow.pkts_toclient: 4
      flow.pkts_toserver: 6
      flow.reason: shutdown
      flow.state: closed
      proto: TCP
      src_ip: 10.16.1.11
      src_port: 54186
      tcp.ack: true
      tcp.fin: true
      tcp.psh: true
      tcp.state: closed
      tcp.syn: true
      tcp.tc_max_regions: 1
      tcp.tcp_flags: 1b
      tcp.tcp_flags_tc: 1b
      tcp.tcp_flags_ts: 1b
      tcp.ts_max_regions: 1
```
After:
```
filter:
    count: 1
    match:
      app_proto: http
      dest_ip: 82.165.177.154
      event_type: flow
      flow.age: 0
      flow.alerted: false
      flow.bytes_toclient: 495
      flow.bytes_toserver: 425
      flow.pkts_toclient: 4
      flow.pkts_toserver: 6
      flow.reason: shutdown
      flow.state: closed
      flow_id: 2019292059448498
      proto: TCP
      src_ip: 10.16.1.11
      src_port: 54186
      tcp.ack: true
      tcp.fin: true
      tcp.psh: true
      tcp.state: closed
      tcp.syn: true
      tcp.tc_max_regions: 1
      tcp.tcp_flags: 1b
      tcp.tcp_flags_tc: 1b
      tcp.tcp_flags_ts: 1b
      timestamp: 2016-07-13T22:42:07.011401+0000
```

### CODE REVIEW

File: createst.py [hunk 1]
```diff
@@ -226,17 +226,30 @@ def get_manipulated_list():
...
     jcontent_list = [json.loads(e) for e in content_list]
     all_content_list = []
     for e in jcontent_list:
-        md = {k: v for k, v in e.items() if k not in skip_fields}
+        md = exclude_nested_fields(e)
+        
         if "event_type" in md and md["event_type"] == "stats":
             continue
-        all_content_list.append(md)
+        all_content_list.append(md)
```
Comment (catenacyber) [inline createst.py:252]: nit typo added space

**Comment by catenacyber** [comment]
timestamp: 2016-07-13T22:42:07.011401+0000should not be readied

File: createst.py [hunk 2]
```diff
@@ -226,17 +226,30 @@ def get_manipulated_list():
...
+        return data
+
     with open(eve_path, "r") as fp:
         content = fp.read()
     content_list = content.strip().split("\n")
     jcontent_list = [json.loads(e) for e in content_list]
     all_content_list = []
     for e in jcontent_list:
-        md = {k: v for k, v in e.items() if k not in skip_fields}
+        md = exclude_nested_fields(e)
```
Comment (catenacyber) [inline createst.py:248]: `skip_fields` should till be used

**Review by catenacyber — CHANGES_REQUESTED** [review]
`skip_fields` should still be used

**Comment by Nancyenos** [comment]
@catenacyber noted ...working on it

**Comment by Nancyenos** [comment]
work continued in #2135

**Pull request closed without merging**